[rospy.client][INFO] 2021-12-06 08:40:26,034: init_node, name[/ss42/duckiebot_visualizer], pid[5260]
[xmlrpc][INFO] 2021-12-06 08:40:26,044: XML-RPC server binding to 0.0.0.0:0
[rospy.init][INFO] 2021-12-06 08:40:26,062: ROS Slave URI: [http://ss42.local:35643/]
[xmlrpc][INFO] 2021-12-06 08:40:26,072: Started XML-RPC server [http://ss42.local:35643/]
[rospy.impl.masterslave][INFO] 2021-12-06 08:40:26,072: _ready: http://ss42.local:35643/
[rospy.registration][INFO] 2021-12-06 08:40:26,086: Registering with master node http://ss42.local:11311/
[xmlrpc][INFO] 2021-12-06 08:40:26,086: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2021-12-06 08:40:26,163: registered with master
[rospy.rosout][INFO] 2021-12-06 08:40:26,165: initializing /rosout core topic
[rospy.rosout][INFO] 2021-12-06 08:40:26,227: connected to core topic /rosout
[rospy.simtime][INFO] 2021-12-06 08:40:26,286: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rosout][INFO] 2021-12-06 08:40:26,448: [/ss42/duckiebot_visualizer] Initialzing.
[rospy.internal][INFO] 2021-12-06 08:40:26,508: topic[/rosout] adding connection to [/rosout], count 0
[rosout][INFO] 2021-12-06 08:40:26,608: [/ss42/duckiebot_visualizer] ~veh_name = ss42 
[rosout][INFO] 2021-12-06 08:40:26,841: [/ss42/duckiebot_visualizer] Initialzed.
[rospy.internal][INFO] 2021-12-06 08:40:32,613: topic[/ss42/ground_projection/lineseglist_out] adding connection to [http://ss42.local:38171/], count 0
[rospy.core][INFO] 2021-12-06 09:22:14,550: signal_shutdown [signal-2]
[rosout][INFO] 2021-12-06 09:22:14,612: [/ss42/duckiebot_visualizer] Shutting down.
[rospy.internal][WARNING] 2021-12-06 09:22:14,882: Unknown error initiating TCP/IP socket to ss42.local:44311 (http://ss42.local:38171/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rosgraph/network.py", line 364, in read_ros_handshake_header
    raise ROSHandshakeException("connection from sender terminated before handshake header received. %s bytes were received. Please check sender for additional details."%b.tell())
ROSHandshakeException: connection from sender terminated before handshake header received. 0 bytes were received. Please check sender for additional details.

[rospy.internal][INFO] 2021-12-06 09:22:15,268: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2021-12-06 09:22:15,272: topic[/ss42/ground_projection/lineseglist_out] removing connection to http://ss42.local:38171/
[rospy.impl.masterslave][INFO] 2021-12-06 09:22:15,281: signal-2
[rospy.core][INFO] 2021-12-06 09:22:15,456: signal_shutdown [atexit]
